packages/engine/paima-rest/src/controllers/AchievementsController.ts

```diff
@@ -9,20 +9,23 @@ import {
   type Player,
   getNftOwner,
 } from '@paima/utils-backend';
-import { getAchievementTypes, getAchievementProgress } from '@paima/db';
+import { getAchievementTypes, getAchievementProgress, getMainAddress } from '@paima/db';
 
 // ----------------------------------------------------------------------------
 // Controller and routes per PRC-1
 
 @Route('achievements')
 export class AchievementsController extends Controller {
   private async game(): Promise<Game> {
-    return { id: 'TODO' };
+    return {
+      id: 'TODO',
+      // TODO: name, version
+    };
   }
 
   private async validity(): Promise<Validity> {
     return {
-      caip2: ENV.CHAIN_ID,
+      caip2: `eip155:${ENV.CHAIN_ID}`,
```

Comment:
This is good enough for now because Paima only supports EVM chains for the main rollup contract, but we should probably create a TODO to address this at some point because this is going to break later this year when we add support for DA layers (which will be a cheaper place for rollup data than our current rollup contract system)